lib/media/track/remotevideotrack.js

```diff
@@ -338,15 +338,20 @@ class RemoteVideoTrack extends RemoteMediaVideoTrack {
   }
 }
 
-function setupDocumentVisibilityTurnOff(removeVideoTrack) {
-  function onVisibilityChanged() {
-    maybeUpdateEnabledHint(removeVideoTrack);
+function isAttachedToDocumentPip(remoteVideoTrack) {
+  if (!('documentPictureInPicture' in window)) {
+    return false;
   }
 
-  documentVisibilityMonitor.onVisibilityChange(1, onVisibilityChanged);
-  return () => {
-    documentVisibilityMonitor.offVisibilityChange(1, onVisibilityChanged);
-  };
+  const pipWindow = documentPictureInPicture.window;
+  if (!pipWindow) {
+    return false;
+  }
+
+  const pipEls = new WeakSet();
+  pipWindow.document.querySelectorAll('video').forEach(el => pipEls.add(el));
```

Comment:
In this context, this is equivalent and simpler
```suggestion
  const pipEls = new WeakSet(pipWindow.document.querySelectorAll('video'));
```